src/codegen/bitfield_unit.rs

```diff
@@ -0,0 +1,83 @@
+#[derive(Copy, Clone, Debug, Default, Eq, Hash, Ord, PartialEq, PartialOrd)]
+pub struct BindgenBitfieldUnit<Storage, Align>
```

Comment:
This definitely needs to be `repr(C)`.

Reply:
Also, I'd recommend maybe to put a double underscore as the prefix for the name, since that's _technically_ reserved in C. Not sure what's @fitzgen's opinion on that.

Reply:
@emilio Nice catch! Thanks. Fixed.